pilot/pkg/xds/endpoints/endpoint_builder.go

```diff
@@ -344,6 +348,15 @@ func (b *EndpointBuilder) BuildClusterLoadAssignment(endpointIndex *model.Endpoi
 		}
 	}
 
+	// If we're an east west gateway, then we also want to send to waypoints
```

Comment:
I think this is pending resolution on the design

Reply:
Yep, good catch. Hoping to close on the design soon, but I suspect we'll remove this

Reply:
Actually, I take this back; I think regardless of what design we pick, the e/w gateway will need to send to waypoints in some capacity. The only TODO is when we decide to send to a waypoint vs. not. I'll add a TODO here, but I wouldn't think it's blocking since none of this is wired up yet